pages/index.js

```diff
@@ -3,33 +3,70 @@ import Form from "@/components/Form/Form.js";
 import ShoppingItem from "@/components/ShoppingItem/ShoppingItem.js";
 import { useState } from "react";
 import { categories } from "@/lib/categoriesData";
+import Image from "next/image";
+
 export default function ShoppingItemsList({
   shoppingItemsWithCategoryColor,
+  backgroundColor,
+  shoppingItems,
   onAddItem,
   onEditItem,
   onDeleteItem,
+  onToggleIsPurchased,
+  purchasedItems,
+  unpurchasedItems,
 }) {
   const [mode, setMode] = useState("");
 
   const [editedItem, setEditedItem] = useState({});
 
+  const [isPurchased, setIsPurchased] = useState("");
+
   function handleChangeMode(mode) {
     setMode(mode);
   }
 
   return (
     <>
       <StyledHeader>
-        <h1>Shopping Car-d</h1>
-      </StyledHeader>
+        <h1>
+          Tot
+          <span style={{ fontSize: 30, color: "green", letterSpacing: "-8px" }}>
+            e
+          </span>
+          &#39;
+          <span style={{ fontSize: 42 }}>A</span>lly
+          <br></br>
+          <span
+            style={{
+              fontSize: 20,
+              color: "green",
+            }}
+          >
+            ...baggin&#39; it.
+          </span>
+        </h1>
 
+        <Image
+          alt="a raw sketch of a tote bag"
+          src={"/images/tote-bag_01.png"}
+          style={{ objectFit: "contain" }}
+          width={60}
+          height={60}
+        />
+      </StyledHeader>
       <main>
-        {shoppingItemsWithCategoryColor.length === 0 && (
-          <StyledNoItemsMessage>
-            There are no items on your shopping list. Add items using the form
-            below.
-          </StyledNoItemsMessage>
+        {mode !== "edit" && mode !== "add" && (
+          <AddItemContainer>
+            {shoppingItemsWithCategoryColor.length === 0 && (
```

Comment:
You want to check here, if there is anything else on the shopping list (e.g. the unpurchased items list):

```suggestion
            {unpurchasedItems.length === 0 && (
```

Reply:
I now have two different messages - one for when there are literally no shopping items on the entire list and one for when all items on the list have been purchased.

Reply:
Ok, but instead of passing `shoppingItemsWithCategoryColor` only for that single purpose down as a prop, you could use `unpurchasedItems` and `purchasedItems` to check for the same. If both of them have a length of 0, there are no shopping items at all.

Reply:
Done.